packages/celotool/src/cmds/account/invite.ts

```diff
@@ -12,39 +14,102 @@ export const describe = 'command for sending an invite code to a phone number'
 
 interface InviteArgv extends AccountArgv {
   phone: string
-  fast: boolean
 }
 
 export const builder = (yargs: Argv) => {
-  return yargs
-    .option('phone', {
-      type: 'string',
-      description: 'Phone number to send invite code,',
-      demand: 'Please specify phone number to send invite code',
-    })
-    .option('fast', {
-      type: 'boolean',
-      default: false,
-      description: "Don't download artifacts, use this for repeated invocations",
-      demand: 'Please specify phone number to send invite code',
-    })
+  return yargs.option('phone', {
+    type: 'string',
+    description: 'Phone number to send invite code,',
+    demand: 'Please specify phone number to send invite code',
+  })
+}
+
+export async function convertToContractDecimals(
+  value: number | BigNumber,
+  contract: StableTokenWrapper
+) {
+  const decimals = new BigNumber(await contract.decimals())
+  const one = new BigNumber(10).pow(decimals.toNumber())
+  return one.times(value)
 }
 
 export const handler = async (argv: InviteArgv) => {
-  console.log(`Sending invitation code to ${argv.phone}`)
+  const phone = argv.phone
+
+  console.log(`Sending invitation code to ${phone}`)
+  await execCmd('gcloud config set project celo-testnet')
```

Comment:
I believe we have a utility function that switches you to the correct project based on the `.env` file

Reply:
Key is only present in celo-testnet so I added a comment